src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -1923,9 +1948,11 @@ def __init__(self, config: BertConfig, *inputs, **kwargs):
     @add_start_docstrings_to_model_forward(BERT_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint=_CHECKPOINT_FOR_QA,
         output_type=TFQuestionAnsweringModelOutput,
         config_class=_CONFIG_FOR_DOC,
+        expected_output="'a nice puppet'",
+        expected_loss=7.41,
```

Comment:
We can reuse the variables you defined at the beginning

Reply:
Yeah, I totally agree